Alignment-INDOT/PR_Twin_Branch_section_alignment.ifc

```diff
@@ -0,0 +1,148 @@
+ISO-10303-21;
+HEADER;
+FILE_DESCRIPTION(('ViewDefinition [Ifc4NotAssigned]'),'2;1');
+FILE_NAME(
+/* name */ 'C:\\Users\\lars\\Downloads\\ggIfcConverter\\PR_Twin_Branch_section_alignment.ifc',
+/* time_stamp */ '2021-03-04T10:18:06',
+/* author */ ('lars'),
+/* organization */ ('Unknown'),
+/* preprocessor_version */ 'ggIfcConverter v0.0.3.0 by Geometry Gym Pty Ltd built 2021-03-04T09:16:31',
+/* originating_system */ 'ggIFCConverter - Converter by Geometry Gym Pty Ltd',
+/* authorization */ 'None');
+
+FILE_SCHEMA (('IFC4X3_RC3'));
+ENDSEC;
+
+DATA;
+#1= IFCCARTESIANPOINT((0.0,0.0,0.0));
+#2= IFCDIRECTION((1.0,0.0,0.0));
+#3= IFCDIRECTION((0.0,1.0,0.0));
+#4= IFCDIRECTION((0.0,0.0,1.0));
+#5= IFCAXIS2PLACEMENT3D(#1,$,$);
+#6= IFCCARTESIANPOINT((0.0,0.0));
+#7= IFCAXIS2PLACEMENT2D(#6,$);
+#8= IFCSIUNIT(*,.LENGTHUNIT.,$,.METRE.);
+#9= IFCSIUNIT(*,.AREAUNIT.,$,.SQUARE_METRE.);
+#10= IFCSIUNIT(*,.VOLUMEUNIT.,$,.CUBIC_METRE.);
+#11= IFCMEASUREWITHUNIT(IFCLENGTHMEASURE(0.304800609601219),#8);
+#12= IFCCONVERSIONBASEDUNIT(#13,.LENGTHUNIT.,'USSurveyFoot',#11);
+#13= IFCDIMENSIONALEXPONENTS(1,0,0,0,0,0,0);
+#14= IFCSIUNIT(*,.PLANEANGLEUNIT.,$,.RADIAN.);
+#15= IFCUNITASSIGNMENT((#12,#14));
+#16= IFCPROJECT('3AmF9rizLFdPhI8nWCsL67',$,'UNKNOWN PROJECT',$,$,$,$,(#51),#15);
+#17= IFCCARTESIANPOINT((0.0,0.0,0.0));
+#18= IFCAXIS2PLACEMENT3D(#17,#4,#2);
+#19= IFCLOCALPLACEMENT($,#18);
+#20= IFCSITE('2PlVuc8pjDywr$PZgmi13a',$,$,$,$,#19,$,$,.COMPLEX.,$,$,$,$,$);
+#21= IFCLOCALPLACEMENT($,#5);
+#22= IFCALIGNMENT('0GLEj6tMb19u3SzxDDGgUo',$,'PR_Twin_Branch_section',$,$,#21,$,$,$);
+#23= IFCRELCONTAINEDINSPATIALSTRUCTURE('0Y93m4$W95IhXby4BL_2BW',$,'Site','Site Container for Elements',(#22),#20);
+#24= IFCALIGNMENTHORIZONTAL('2mv6_WjB5EnBUjc2pdivhk',$,$,$,$,#21,#57,0.0,(#));
```

Comment:
Lars,
The (#) in #40 is incorrect. 
I think this file was produced with an older version of the library you use.
I believe it has now been corrected in the latest library.